memsql-entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+if [ "$1" = "memsqld" ]; then
+    # Start up the cluster
+    memsql-ops start
+    memsql-ops memsql-start --all
+    memsql-ops memsql-list
```

Comment:
Starts the cluster. Changed away from init.d. Used the postgres image as an example. Still lets you run other commands against the image.

https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#entrypoint

Realize ops probably starts the cluster, but made it explicit and listed the nodes.